Gemfile

```diff
@@ -35,15 +20,6 @@ gem 'select2-rails'
 # Use Unicorn as the app server
 # gem 'unicorn'
 
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
-
-# Use redis for caching
-gem 'redis'
-gem 'redis-namespace'
-gem 'redis-rails'
-gem 'redis-rack-cache'
```

Comment:
I've dropped redis for session storage. I thought we could bring it back in when we start caching content.